### PR TITLE
Feature/app layout

### PR DESCRIPTION
**Pull request by sarahwylie**
- Pages have preliminary layout
- App.js has preliminary layout
- Header and Footer are started
- Styling has begun in SASS

### CODE REVIEW

File: Develop/client/src/App.js [hunk 1]
```diff
@@ -1,24 +1,55 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <ApolloProvider client={client}>
+      <Header />
```
Comment (JideOgun) [inline Develop/client/src/App.js:36]: The header is not being imported

File: Develop/client/src/App.js [hunk 2]
```diff
@@ -1,24 +1,55 @@
...
+    <Router>
+      <>
+        <Navbar />
```
Comment (JideOgun) [inline Develop/client/src/App.js:39]: Navbar is not being imported

File: Develop/client/src/pages/Outdoor.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Outdoor = () => {
+
```
Comment (JideOgun) [inline Develop/client/src/pages/Outdoor.js:4]: same here

**Review by JideOgun — CHANGES_REQUESTED** [review]
Logo looks a bit compressed

**Comment by nicnolen** [comment]
Instead of .scss, use .sass. It keeps the code DRY. The only difference is dont use semicolons or curly braces. Also make sure this is in the assets.sass folder or the css will not be created correctly

**Pull request closed without merging**